webpack.config.ts

```diff
@@ -118,7 +118,7 @@ const modeConfigs: {
             priority: 1,
             transform: (content, fileName) => {
               // Inject devtools in every static HTML file
-              if (fileName.endsWith(".html")) {
+              /*if (fileName.endsWith(".html")) {
```

Comment:
did you mean to leave this dead?

Reply:
Yes. React-devtools is broken in manifest v3 because the page attempts to load a script on a different URL which is against policy. I have commented out the code and made an issue: https://github.com/PelagusWallet/pelagus-extension/issues/16